spec/controllers/projects_controller_spec.rb

```diff
@@ -254,6 +254,43 @@
       before{ get :show, permalink: project.permalink, project_post_id: project_post.id, locale: :pt }
       it("should assign update to @update"){ expect(assigns(:post)).to eq(project_post) }
     end
+
+    context "@supporters" do
+      let(:project) { create(:project) }
+
+      before do
+        allow_any_instance_of(Project).to receive(:recurring?).and_return(true)
+      end
+
+      context "when there are paid subscriptions for the project" do
+        before do
+          user_1 = create(:user, name: 'fulano')
```

Comment:
Avoiding the expect part, you still can turns this test more readable renaming the variables with the user name.